lms/templates/navigation.html

```diff
@@ -124,15 +124,21 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
         % endif
       </%block>
       % if not settings.FEATURES['DISABLE_LOGIN_BUTTON']:
+          % if settings.FEATURES.get('ENABLE_COURSE_DISCOVERY'):
```

Comment:
I'm fine punting the changes, but please do be sure to capture them in a backlog somewhere.  @pbaruah asked if I could do a review of this PR, but I can't really comment on the Sass changes, so you may want to find someone else -- perhaps another guy from EE?